src/main/java/org/fundacionjala/coding/maria/Kata13.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.coding.maria;
+
+/**
+ * Kata 13, validate number.
+ */
+public class Kata13 {
+    private static final int TEN = 10;
+    private static final int THREE = 10;
+    /**
+     * @param number string number to validate
+     * @return boolean if the number is valid
+     */
+    public boolean validate(String number) {
+        String[] numberArray = number.split("");
+        int total = 0;
+        for (int i = 0; i < numberArray.length - 1; i += 2) {
+            total += Integer.parseInt(numberArray[i]);
+            total += Integer.parseInt(numberArray[i + 1]) * THREE;
+        }
+        total = total % total == 0 ? 0 : total - (total % TEN);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make sure "total" can't be zero before doing this modulation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3518)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Correct one of the identical sub-expressions on both sides of operator "%" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1764)